relay-server/src/envelope.rs

```diff
@@ -106,6 +106,8 @@ pub enum ItemType {
     ProfilingSession,
     /// Profiling data
     ProfilingTrace,
+    /// Stacktrace event payload encoded in JSON
+    Stacktrace,
```

Comment:
A general question on the naming of this: Does this still contain profiling/specto-specific payloads? If so, it would be great if we could reflect this in the name of this item type somehow. 

In parts, my thinking here is that we also have a [stack trace schema](https://develop.sentry.dev/sdk/event-payloads/stacktrace/) in Sentry's error events. It would be good if we either separate the two clearly, including by using different terminology, or alternatively use the same schema. Choosing the second option is clearly a larger discussion.

Same applies to all other references in this PR, such as the Kafka topic assignment.

Reply:
It does not contain anything Specto specific, it's only profiling related now. You're not the only one mentioning this so I decided to change the name to "profile". I think it's a better name overall since it's easy to think "profiling team manages profiles" and it doesn't collide with other names in other repos (except maybe user profiles but it's so unrelated, I think it's OK).

Reply:
That said, it probably still makes sense somewhat aligning the fields so SDKs and backend code does not have to deal with two stacktrace formats. Particularly if you plan on forwarding this to symbolicator.

Reply:
I guess we do have the timing information for each address, so we can't 100% match that format. We'll use a superset of that format, I'll need to transform what I get anyway to send the addresses to symbolicator but keep the original format.

It's going to look like that:
```json
{
    "samples": [
      {
       "timestamp": <some duration from start of transaction>,
        "thread_id": "<some uint64 thread ID>",
        "frames": [
          { "instruction_addr": "0x7fff5bf3456c" },
          { "instruction_addr": "0x7fff5bf346c0" }
          ...
       ],
      },
      ...
    ]
}
```